resources/views/partials/item/wishlist-button.blade.php

```diff
@@ -0,0 +1,8 @@
+<button v-if="$root.loggedIn" class="p-3" @click="toggle">
+    <x-heroicon-s-heart v-if="isWishlisted({{ $productId }})" class="w-5 h-5 text-ct-error hover:opacity-80" />
+    <x-heroicon-o-heart v-else v-bind:class="{ 'text-ct-primary': isOpen, 'text-ct-border hover:text-ct-primary': !isOpen }" class="w-5 h-5 transition" />
+</button>
+
+<button v-else class="p-3" @click="toggle">
+    <x-heroicon-o-heart class="w-5 h-5 transition" v-bind:class="{ 'text-ct-primary': isOpen, 'text-ct-border hover:text-ct-primary': !isOpen }" />
```

Comment:
`class` here before the `v-bind:class`, whereas above it's the other way around?